Removed info about applications.commands scope that is no longer relevant (#5669)
hammerandchisel/discord-api-docs: docs/Getting_Started.md

> info
> The URL generator helps create an installation link with the scopes and permissions your app needs to function. You can use the link to install the app onto your server, or share it with others so they can install it on their own.

> For now, we’ll just add one scope:
> - `bot` is to enable your bot user. After you click bot, you can also add different user permissions to the bot. For now, just check **Send Messages**.
> 

Reviewer: ```suggestion
For now, we’ll just add the **`bot` scope**, which adds your bot user.

After you select `bot`, you can also select different permissions for your bot. For now, just check **Send Messages**.

```
